R-package/R/lgb.Dataset.R

```diff
@@ -127,7 +195,7 @@ Dataset <- R6::R6Class(
         cnames <- colnames(private$raw_data)
       }
 
-      # set feature names if not exist
+      # set feature names if they don't exist
```

Comment:
This change does not look directly related to this feature. Can you please open a separate pull request for this? We have a strong preference in this project for tightly-focused pull requests that are only about one thing, wherever possible.

Reply:
Reverted.

Reply:
thank you